src/coord/src/catalog/builtin.rs

```diff
@@ -1157,6 +1157,14 @@ pub static MZ_CLUSTER_REPLICAS_STATUS: Lazy<BuiltinTable> = Lazy::new(|| Builtin
         .with_column("last_update", ScalarType::TimestampTz.nullable(false)),
 });
 
+pub static MZ_CLUSTER_REPLICA_HEARTBEATS: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
+    name: "mz_cluster_replica_heartbeats",
```

Comment:
The status table is named "mz_cluster_replica**s**_status". I used the plural form because then it's just "mz_cluster_replica" with a suffix. I'm now wondering if we should rename that to "mz_cluster_replica_status" to make it consistent with the new heartbeats table (and possible future ones).

Reply:
If we were to apply the naming convention exactly, that other one should be called `mz_cluster_replica_statuses`, I guess!

Reply:
> But I feel like mz_cluster_replica_ is somewhat better English.
I agree!